cmd/anonymizer/app/uiconv/reader_test.go

```diff
@@ -99,3 +102,19 @@ func TestReader_TraceInvalidJson(t *testing.T) {
 	assert.Equal(t, 0, r.spansRead)
 	assert.Equal(t, true, r.eofReached)
 }
+
+func TestReader_GetWDError(t *testing.T) {
+	inputFile := "fixtures/trace_non_existent.json"
+
+	wd, err := os.Getwd()
+	require.NoError(t, err)
+	err = os.Chmod(wd, 0000)
+	require.NoError(t, err)
+	defer os.Chmod(wd, 0755)
```

Comment:
This is clever, but it's conceivable that the tests might run on a read-only source dir where chmod won't work. If we can get rid of Getwd per my comment above, let's do that instead.